src/styles/main.scss

```diff
@@ -16,6 +16,7 @@
 
 html {
   width: 100vw;
+  scroll-padding-top: 57px;
```

Comment:
On mobile, there's a button that lets you jump to the form, but because the header floats above everything else on the page, the form gets cut off.  Adding the padding here makes the anchor tag jump to the right place to display the entire form.

Reply:
FYI @sjwmoveon, new scroll-padding-top values should be used for compatibility with the new header if this pull request is merged. I've copied the correct values below. (I suppose only the first one applies, if the button to jump will only ever appear on mobile.) Thank you!

```
  html {
    scroll-padding-top: 145px;
  }
  @media (min-width: 768px) {
    html {
      scroll-padding-top: 149px;
    }
  }
  @media (min-width: 1025px) {
    html {
      scroll-padding-top: 102px;
    }
  }
```